tests/functional/test_contract_container.py

```diff
@@ -66,3 +66,16 @@ def test_deploy_proxy(
     # Show we get the implementation contract type using the proxy address
     implementation = chain.contracts.instance_at(proxy.address)
     assert implementation.contract_type == vyper_contract_instance.contract_type
+
+
+def test_source_path_in_project(project_with_contract):
+    path = (
+        project_with_contract.contracts_folder
+        / project_with_contract.contracts["ApeContract0"].source_id
+    )
+    assert path
+    assert project_with_contract.get_contract("ApeContract0").source_path == path
```

Comment:
```
>       assert project_with_contract.get_contract("ApeContract0").source_path == path
E       AssertionError: assert None == PosixPath('/tmp/tmp5qew_fm2/ApeProject/contracts/ApeContract0.json')
```
😅

Reply:
somehow I need this to make it work: `project_with_contract.get_contract("ApeContract0").source_path`

Reply:
You could replace `ApeContract0.json` with a `ContractType` JSON.